app/src/main/java/io/github/wztlei/wathub/MainApplication.java

```diff
@@ -1,5 +1,7 @@
 package io.github.wztlei.wathub;
 
+//test
```

Comment:
Remove this comment please

Reply:
I think that is removed after I pulled from master into my branch